src/main.rs

```diff
@@ -13,8 +13,12 @@ use regex::Regex;
 
 use vmtest::{Config, Target, Ui, VMConfig, Vmtest};
 
+const HELP_ENV_VARS: &str = r#"Environment variables:
+  VMTEST_NO_UI    Set to disable UI  [default: unset]
+"#;
+
 #[derive(Parser, Debug)]
-#[clap(version)]
+#[clap(version, disable_colored_help=true, after_help=HELP_ENV_VARS)]
```

Comment:
clap supports both parameter and env variable by setting `env` in the argument derive. https://stackoverflow.com/a/73531388

This means that the value of `args.dry_run` could be set either from the CLI with `--dry-run` or from the environment with `DRY_RUN=1`, not exactly what you are trying to do, but mentioning htis in case.

Reply:
I was trying to avoid needing to thread the variable through a bunch of the code (which would require a larger refactor). Seemed ok enough to just bypass with env var :P